### ISSUE TITLE
Feature request: Set the max size of the tab in tab bar

### ISSUE THREAD

**Issue by ReporterX** [notepad-plus-plus/notepad-plus-plus#5563]
For tabs with long file names, they take up so much space.
I open 15 tabs. It takes up 7 rows in multi-line mode. 

You can see 2 tabs occupy the whole line.
it wastes a lot of screen space because the third tab could not fit in.

![screenshot](https://user-images.githubusercontent.com/8235338/56502715-e4f63400-6545-11e9-83aa-d769efda404e.jpg)

**Comment by neviln** [comment]
YES PLEASE!!! I went searching to see how I can do this and found so many topics on multiple different forums asking about this exact thing. Then I saw this post from 2019 and am just surprised it hasn't been implemented yet. Any updates would be greatly appreciated. Thanks

**Comment by freezer2022** [comment]
Duplicate of:
- https://github.com/notepad-plus-plus/notepad-plus-plus/issues/3332
